app/src/pages/settings/AnnotationConfigDialog.tsx

```diff
@@ -0,0 +1,424 @@
+            mode === "new" ? "New Annotation Config" : "Edit Annotation Config"
+          }
+          variant="compact"
+          bodyStyle={{ padding: 0 }}
+          extra={
+            <Button
+              onPress={close}
+              size="S"
+              css={css`
+                & {
+                  padding-left: var(--ac-global-dimension-size-50) !important;
+                  padding-right: var(--ac-global-dimension-size-50) !important;
+                }
+              `}
+            >
+              <Icon svg={<Icons.CloseOutline />} />
+            </Button>
+          }
+        >
+          <Form
+            onSubmit={(e) => {
+              handleSubmit((data) => {
+                onSubmit(data, close);
+              })(e);
+            }}
+          >
+            <View
+              minWidth="200px"
+              padding="size-200"
+              maxHeight="600px"
+              overflow="auto"
+            >
+              <Flex
+                direction="column"
+                gap="size-200"
+                className="new-annotation-dialog-container"
+              >
+                <Controller
+                  name="name"
+                  control={control}
+                  rules={{
+                    required: "Name is required",
+                  }}
+                  render={({ field, fieldState: { error } }) => (
+                    <TextField {...field} isInvalid={!!error}>
+                      <Label>Annotation Name</Label>
+                      <Input placeholder="correctness" />
+                      <FieldError>{error?.message}</FieldError>
+                    </TextField>
+                  )}
+                />
+                <Controller
+                  name="description"
+                  control={control}
+                  render={({ field }) => (
+                    <TextField
+                      {...field}
+                      value={field.value ?? undefined}
+                      css={css`
+                        & .react-aria-TextArea {
+                          resize: vertical;
+                          transition: none;
+                        }
+                      `}
+                    >
+                      <Label>Description</Label>
+                      <TextArea
+                        rows={2}
+                        placeholder="A description of the annotation configuration"
+                      />
+                    </TextField>
+                  )}
+                />
+                <Controller
+                  control={control}
+                  name="annotationType"
+                  render={({ field }) => (
+                    <RadioGroup
+                      {...field}
+                      aria-label="Type"
+                      data-testid="type-picker"
+                      direction="column"
+                      isReadOnly={mode === "edit"}
+                    >
+                      <Label>Type</Label>
```

Comment:
```suggestion
                      <Label>Annotation Type</Label>
```

Probably requires a bit of description (maybe on the values?)